src/ResourceManagement/CosmosDB/CassandraTableImpl.cs

```diff
@@ -411,7 +411,7 @@ public CassandraTableImpl WithClusterKey(string name, string orderBy)
 
         public CassandraTableImpl WithoutClusterKey(string name)
         {
-            this.createUpdateParameters.Resource.Schema?.ClusterKeys?.Remove(this.createUpdateParameters.Resource.Schema?.ClusterKeys?.Single(element => element.Name == name));
+            this.createUpdateParameters.Resource.Schema?.ClusterKeys?.Remove(this.createUpdateParameters.Resource.Schema?.ClusterKeys?.FirstOrDefault(element => element.Name == name));
```

Comment:
Or should we remove all if duplicate name found?

BTW, if remove all, this might be simpler to understand `list.Where(...).ToList();`

Reply:
I have thought about the question, and think only remove one is more flexible, it can be call multiple times to remove all. (actually, duplicate name is rare to see)